training/README.md

```diff
@@ -0,0 +1,247 @@
+## How to Train Object Detector Model Using Your Own Data
```

Comment:
I have rearranged the general README template to go along with the model. For example,
`Run the setup script` comes before `Prepare data for training`. This is because the user needs to upload the data first and then proceed to annotation whose steps are included in `Prepare data for training`. From there to customize hyperparamaters

Reply:
Ok sure